test/config/rewriters/OasFeatureRewriterSpec.scala

```diff
@@ -24,51 +24,59 @@ import support.UnitSpec
 
 class OasFeatureRewriterSpec extends UnitSpec with MockAppConfig {
 
-  private def setupCheckAndRewrite(oasFeatureEnabled: Boolean, oasFeatureEnabledInProd: Boolean): (CheckRewrite, Rewriter) = {
+  private def setupCheckAndRewrite(oasFeatureEnabled: Boolean, oasFeatureReleasedInProd: Boolean): (CheckRewrite, Rewriter) = {
     MockAppConfig.featureSwitches returns Configuration(
       "oasFeature.enabled"                -> oasFeatureEnabled,
-      "oasFeature.released-in-production" -> oasFeatureEnabledInProd
+      "oasFeature.released-in-production" -> oasFeatureReleasedInProd
     )
 
     val rewriter = new OasFeatureRewriter()(mockAppConfig)
     rewriter.rewriteOasFeature.asTuple
   }
 
-  "check and rewrite" should {
-    "indicate that it wants to rewrite the file" in {
-      val (check, _) = setupCheckAndRewrite(oasFeatureEnabled = true, oasFeatureEnabledInProd = true)
-      val result     = check("1.0", "oasFeature")
-      result shouldBe true
+  "check" should {
+    "always return true" in {
+      List(
+        (true, true),
+        (true, false),
+        (false, true),
+        (false, false)
+      ).foreach { case (enabled, enabledInProd) =>
+        val (check, _) = setupCheckAndRewrite(oasFeatureEnabled = enabled, oasFeatureReleasedInProd = enabledInProd)
+
+        val result = check("1.0", "any-file.yaml")
+        result shouldBe true
+      }
     }
+  }
 
-    "rewrite" when {
-
-      val yaml =
-        """
-          |summary: Retrieve Business Details
-          |description: |
-          |  This endpoint enables you to retrieve existing business details.
-          |  A National Insurance number and tax year must be provided.
-          |
-          |  ### Test data
-          |  {{#if (testOnly "oasFeature")}}
-          |  <p>Scenario simulations using Gov-Test-Scenario headers ARE ONLY AVAILABLE IN the sandbox environment.</p>
-          |  {{else}}
-          |  <p>Scenario simulations using Gov-Test-Scenario headers are only available in the sandbox environment.</p>
-          |  {{/if}}
-          |
-          |tags:
-          |  - Employment Expenses
-          |""".stripMargin
-
-      "the feature isn't enabled" in {
-        val (_, rewrite) = setupCheckAndRewrite(oasFeatureEnabled = false, oasFeatureEnabledInProd = false)
+  "rewrite" should {
+
+    val yaml =
+      """
+        |summary: Retrieve Employment Expenses
```

Comment:
```suggestion
        |summary: Retrieve Business Details
```

For all the occurrences of `Retrieve Employment Expenses`